__tests__/api.test.js

```diff
@@ -93,6 +93,106 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  test("should respond with 200 status", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("should respond with an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        expect(body).toHaveProperty("articles", expect.any(Array));
+      });
+  });
+  test("array should contain object with correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(body).toHaveProperty("articles", expect.any(Array));
+        expect(typeof articles).toBe("object");
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(String));
+        });
+        expect(articles.length).toEqual(13);
+      });
+  });
+  test("should be ordered by date descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should return 200 status", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("should respond with an array attached to the key of comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        expect(body).toHaveProperty("comments", expect.any(Array));
+      });
+  });
+  test("should respond with an array of comments containing objects with the correct properties and right length", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toEqual(11);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+        });
+      });
+  });
+  test("should respond with an error message comments not found when no comments are found on a happy path", () => {
```

Comment:
Should this be an error? Is it normal for a post to potentially have no comments?